GliaWidgets/View/Chat/ChatView.swift

```diff
@@ -338,4 +360,13 @@ extension ChatView: UITableViewDelegate {
     public func scrollViewWillBeginDragging(_ scrollView: UIScrollView) {
         endEditing(true)
     }
+
+    func scrollViewDidScroll(_ scrollView: UIScrollView) {
+        let chatEnd = scrollView.contentSize.height - scrollView.frame.size.height
+        if scrollView.contentOffset.y + 10 >= chatEnd {
```

Comment:
Should probably use `if scrollView.contentOffset.y + scrollView.contentInset.top >= chatEnd`
Would also rename `chatEnd` to something like `chatBottomOffset`

Reply:
Good point about taking `contentInset` into consideration, haven't thought of that. Thanks

Reply:
I'd leave `+10` though as well so it detects not just pixel-perfect scrolling to the bottom, but even a very close one.